homeassistant/components/elevenlabstts/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domain": "elevenlabstts",
```

Comment:
Can you rename integration ot just `elevenlabs`

Reply:
Changed it :)